DOM_creation/Lesson3/main.js

```diff
@@ -0,0 +1,22 @@
+const ul = document.getElementById("js-ul");
+
+const a1 = document.createElement("a");
+a1.href = "a1.html";
+
+const a2 = document.createElement("a");
+a2.href = "a2.html";
+
+const img1 = document.createElement("img");
+img1.src = "/img/bookmark.png";
+
+const img2 = document.createElement("img");
+img2.src = "/img/message.png";
+
+const array = [{a: a1, img: img1, text: "a1"}, {a: a2, img: img2, text: "a2"}];
+
+array.forEach(e =>{
```

Comment:
Q
Can you tell me why you chose forEach for loop processing?

Reply:
As I originally created "array", so I though using forEach would simplify the code to loop through the array to add element to ul/li.

Reply:
Thanks for the answer.

For Each is fine, but I wanted to know if there is a reason for using for Each, 
as `for of` is faster and better suited for heavy load processing such as DOM manipulation.

Reply:
Mai-san,
Thank you for introducing me for of method. I did not know that there is such a method for iteration. 
I will try use the method for other exercise